bin/generate-tiles

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# For backward compatibility, allow both PG* and POSTGRES_* forms,
+# with the non-standard POSTGRES_* form taking precedence.
+# An error will be raised if neither form is given, except for the PGPORT
+export PGHOST="${POSTGRES_HOST:-${PGHOST?}}"
+export PGDATABASE="${POSTGRES_DB:-${PGDATABASE?}}"
+export PGUSER="${POSTGRES_USER:-${PGUSER?}}"
+export PGPASSWORD="${POSTGRES_PASSWORD:-${PGPASSWORD?}}"
+export PGPORT="${POSTGRES_PORT:-${PGPORT:-5432}}"
+
+# List of postgres servers
+if [[ -z "${PGHOSTS_LIST}" ]] # "xxx.xxx.xxx.xxx&host=xxx.xxx.xxx.xxx&host=..."
+then
+  export HOST_COUNT=1
+  export PGHOSTS="${PGHOST}"
+else
+  export HOST_COUNT=`awk -F"&" '{print NF}' <<< "${PGHOSTS_LIST}"`
+  export PGHOSTS="${PGHOSTS_LIST}"
+fi
+
+export NO_GZIP=${NO_GZIP:-1}
+if [[ ${NO_GZIP} ]] && [[ ! ${NO_GZIP} == "0" ]] && [[ ! ${NO_GZIP} == "true" ]]
+then
+  export NO_GZIP=1
+fi
+
+export KEY=${KEY:-1}
+if [[ ${KEY} ]] && [[ ! ${KEY} == "0" ]] && [[ ! ${KEY} == "true" ]]
+then
+  export KEY=1
+fi
+
+export FUNCZXY="${FUNCZXY:-getmvt}"
+export COPY_CONCURRENCY=${COPY_CONCURRENCY:-1}  # number of CPUs per postgres server
+export MAXCONNECTIONS=${MAXCONNECTIONS:-${COPY_CONCURRENCY}}
+export ALL_STREAMS=$(( MAXCONNECTIONS * HOST_COUNT ))
+
+export EXPORT_DIR=${EXPORT_DIR:-/export}
+export MBTILES_FILE=${MBTILES_FILE:-tiles.mbtiles}
+export RENDER_SCHEME=${RENDER_SCHEME:-pyramid}
+export RETRY=${RETRY:-2}
+export BBOX=${BBOX:-"-180.0,-85.0511,180.0,85.0511"}
+export TIMEOUT=${TIMEOUT:-1800000}
+export MIN_ZOOM=${MIN_ZOOM:-0}
+export MAX_ZOOM=${MAX_ZOOM:-14}
+
+export PGQUERY="pgquery://?database=${PGDATABASE}&host=${PGHOST}&port=${PGPORT}&username=${PGUSER}&password=${PGPASSWORD}&funcZXY=${FUNCZXY}&testOnStartup=false&maxpool=${MAXCONNECTIONS}&nogzip=${NO_GZIP}&key=${KEY}"
```

Comment:
testOnStartup is a highly useful thing to have in many cases -- lets make that into a var too, with default = true

Reply:
`TEST_ON_STARTUP` var added.

Reply:
I had a similar problem as nyurik/tilelive-pgquery#1 Maybe set default to false?

Reply:
i fixed it - the issue was that if the value was not set, it worked correctly. But if you do set it, it has to be a proper `z/x/y` - and i had a bug in parsing it. Fixed in 0.7.3